comment-service/src/main/java/com/farmted/commentservice/controller/CommentController.java

```diff
@@ -0,0 +1,40 @@
+package com.farmted.commentservice.controller;
+
+import com.farmted.commentservice.domain.Comment;
+import com.farmted.commentservice.service.CommentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/comments")
+public class CommentController {
+    @Autowired
+    private CommentService commentService;
+
+    @GetMapping
+    public List<Comment> getAllComments() {
+        return commentService.getALlComments();
+    }
+
+    @GetMapping("/{id}")
+    public Comment getCommentById(@PathVariable Long id) {
+        return commentService.getCommentById(id);
+    }
+
+    @PostMapping
+    public Comment createComment(@RequestBody Comment comment) {
+        return commentService.createComment(comment);
+    }
```

Comment:
```suggestion 
@PostMappring("commets")
public Comment createComment(
@RequestHeader("UUID") String memberUuid,
@PathVariable(value="board_uuid") String boardUuid
@RequestBody CommentCreateRequestDto commentDto) {
        return commentService.createComment(memberUuid, boardUuid, commentDto);
    }
```

위 내용을 적용하면 변화된 부분과 같이 댓글이 작성될 때 누가 작성했는지("memberUuid")는 Header값에서  어느 게시글("boardUuid")에 작성했는지 PathVariable을 사용해 받는 코드가 됩니다. 이 값을 같이 저장해주시면 됩니다!